shared-data/default-theme/html/jsapi/global/eventlog.js

```diff
@@ -61,11 +61,15 @@ EventLog.poll = function() {
   // 3) Other tabs may rely on us putting things in localStorage that we
   //    ourselves don't care about.
   //
+  //time for server to wait for new events
+  var waittime = 30;
+  //extra time to account for processing time on backend
+  var buffer = 5;
   EventLog.request({
     since: EventLog.last_ts,
     gather: (EventLog.last_ts < 0) ? 0.2 : 1.0,
-    wait: 30,
-    _timeout: (Mailpile.ajax_timeout * 3)
+    wait: waittime,
+    _timeout: (waittime+buffer)*1000
```

Comment:
So it seems like `wait` is in seconds and `_timeout` is in milliseconds. Couldn't they be measured in the same unit so you don't need to do this conversion?

Reply:
Unfortunately the _timeout is passed to the jquery $.ajax() function which requires the timeout in milliseconds, but the server requires the requested wait in seconds.